test/suite/test_backup01.py

```diff
@@ -187,7 +188,26 @@ def test_checkpoint_delete(self):
         self.assertRaisesWithMessage(wiredtiger.WiredTigerError,
             lambda: self.session.checkpoint("name=three,drop=(two)"), msg)
         self.session.checkpoint()
+
+        # Need to pause a couple seconds; checkpoints that are assigned the same timestamp as
+        # the backup will be pinned, even if they occur after the backup starts.
+
+        time.sleep(2)
+        # N.B. This test is temporarily disabled because deleting checkpoints during backup
+        # can corrupt the backup.
+        #self.session.checkpoint("name=four")
+        #self.session.checkpoint("drop=(four)")
+        #self.assertRaises(wiredtiger.WiredTigerError,
+        #    lambda: self.session.open_cursor(
+        #    self.objs[0][0], None, "checkpoint=four"))
```

Comment:
Uncomment this test case. It was re-enabled as part of the final fix in WT-6118.

Reply:
Done.